plugins/modules/aci_epg_to_domain.py

```diff
@@ -75,6 +75,20 @@
     - Name of the VMM Domain Enhanced Lag Policy.
     type: str
     aliases: [ lag_policy ]
+  vmm_uplink_active:
+    description:
+    - A list of active uplink IDs.
+    - The order decides the order in which active uplinks take over for a failed uplink.
+    - At least one active uplink must remain specified in list when a active uplink is previously configured.
+    - Not specifying 'vmm_uplink_active' and 'vmm_uplink_standby' will remove a configured child object ( fv:UplinkOrderCont ).
```

Comment:
Why do you say this? Do you actually check and remove this object?

Reply:
I do some tests to check this behaviour. Let's discuss in our next sync what I exactly do and why? Perhaps you have different ideas.

Reply:
removed the comment and tests, because in earlier than 4.2 versions the child object does not exist I cannot pass it by default as child classes list.